src/transformers/tokenization_bart.py

```diff
@@ -53,40 +53,64 @@ def prepare_seq2seq_batch(
         max_target_length: Optional[int] = None,
         padding: str = "longest",
         return_tensors: str = "None",
+        truncation=True,
         **kwargs,
     ) -> BatchEncoding:
         r"""
         Prepare a batch that can be passed directly to an instance of :class:`~transformers.BartModel`.
 
         Args:
-            src_texts (:obj:`List[str]`):
-                List of input texts.
-            tgt_texts (:obj:`List[str]`, `optional`):
-                List of target texts.
+            src_texts: (:obj:`list`):
+                list of documents to summarize or source language texts
+            tgt_texts: (:obj:`list`, `optional`):
+                list of tgt language texts or summaries.
```

Comment:
The type annotations here were better before. The docstrings should not have abbreviations (and start with a capital and end with a full stop nit).

Reply:
Aah, I blindly copy pasted, will make the changes. Also can you tell me the where the doc error is coming from ?

Reply:
You're missing new lines before your lists I'd say.